discover/discover.go

```diff
@@ -139,12 +139,10 @@ func ConvertPolicy(o Options) error {
 
 // Policy discovers Cilium or KubeArmor policies
 func Policy(o Options) error {
-	if o.Policy == "cilium" {
-		o.Policy = "network"
-	} else if o.Policy == "kubearmor" {
-		o.Policy = "system"
+	if o.Class == "apllication" {
+		o.Class = "system"
 	} else {
-		log.Error().Msgf("Policy type not recognized.\nCurrently supported policies are cilium and kubearmor\n")
+		log.Error().Msgf("Policy type not recognized.\nCurrently supported policies are application and network\n")
```

Comment:
Currently it will show an error if we applied class as network because we are checking only for o.Class=="application". Please add a condition to show the error message only in case o.Class != "application" || o.class != "network"